lib/public/services/detectors/detectorsProvider.js

```diff
@@ -35,6 +35,13 @@ export class DetectorsProvider extends Observable {
                 Success: (detectors) => detectors.filter(({ name }) => name !== TST_DETECTOR_NAME),
             }))
             .build();
+        this._dpl$ = new ObservableData(RemoteData.notAsked());
+        this._physicalDpl$ = ObservableData.builder()
```

Comment:
I think that this should go to another provider, probably dplDetectorsProvider

Reply:
applied